public/locales/en/translation.json

```
@@ -0,0 +1,178 @@
+    "landingAboutUsContent1": "The content on",
+    "landingAboutUsContent2": "and",
+    "landingAboutUsContent3": ", is being made available under a collaborative arrangement with multiple partners and",
+    "landingAboutUsContent4":"; who is the localization and technology partner.",
+    "landingAboutUsContentTitle":	"Content",
+    "landingAboutUsTechnologyTitle":	"Technology",
+    "landingAboutUsTechnologyMsg":"BCS team has developed this platform inspired by similar initiatives in other countries. A cloud based, API driven, Biblical Computing engine - VachanEngine is the back-end of this. These Vachan APIs can be made available for digital content delivery on request.",
+    "landingAboutUsContactUs":	"Contact Us",
+    "landingAboutUsGithubRelease":"GitHub Release Notes.",
+    "commonClose":	"Close",
+    "landingPlayStoreToolTip":	"Get the app on the Google Play Store",
+    "landingWelcomeHeading":	"Welcome to",
+    "landingWelcomeMessage1":	"and the companion" ,
+    "landingWelcomeMessage2":	"app is a premier Scripture Engagement website in Indian Languages!",
+    "landingWelcomeMessage3": "So what is Scripture engagement? It is a way of studying the Bible with resources and tools to assist you understand the Bible.With a host of commentaries, videos, audio Bibles and reading plans in your heart language, our desire is that you will find this website to be a place where you can interact with Scripture, find resources to understand it, journal your spiritual growth and enjoy developing a growing relationship with God.",
+    "landingWelcomeMessage4":	"The",
+    "landingWelcomeMessage5":	"companion app enables you take your Bible and your Notes with you wherever you go!",
+    "landingFooterFeedbackBtn":	"Feedback",
+    "landingCompanyYear": "© 2020 ",
+    "landingCompanyMob":	"BCS",
+    "landingCompanyDesktop":	"Bridge Connectivity Solutions",
+    "landingLangBarToolTip":	"Click to read the Bible in" ,
+    "landingFooterAboutUsBtn": "About Us",
+    "loginEmailPlaceHolder":	"Email Address",
+    "loginSignInGoogleBtn":	"SIGN IN WITH GOOGLE",
+    "loginPasswordPlaceHolder": "Password",
+    "loginSignOutBtn":	"Sign Out",
+    "loginSignInWarning":	"Sign in to use this feature",
+    "loginForgotPswd":	"Forgot Password?",
+    "loginSignUpMsg":	"Don't have an account? Sign Up",
+    "loginSignUpTitle":	"Sign Up Title",
+    "loginSignUpBtn":	"SIGN UP",
+    "loginSignInMsg":	"Already have an account? Sign in",
+    "loginBackSignIn":	"Back to Sign in",
+    "loginPswdReset":	"Enter your e-mail address and we'll send you a link to reset your password",
+    "loginSubmit":	"Submit",
+    "studyNoBookMarks":	"No bookmarks added",
+    "studyCommIntroTo":	"Introduction to ",
```

Add interpolation here and in hindi file 
```suggestion
    "studyCommIntroTo": "Introduction to {{book}}",
```